src/core/Akka/Dispatch/Mailbox.cs

```diff
@@ -85,14 +85,14 @@ internal static class MailboxStatus
          * ANOTHER THING, IMPORTANT:
          *
          * ActorCell.start() publishes actorCell & self to the dispatcher, which
-         * means that messages may be processed theoretically before self’s constructor
+         * means that messages may be processed theoretically before self's constructor
          * ends. The JMM guarantees visibility for final fields only after the end
          * of the constructor, so safe publication requires that THIS WRITE BELOW
          * stay as it is.
          */
         private volatile ActorCell _actor = null;
         private volatile SystemMessage _systemQueueDoNotCallMeDirectly = null; // null by default
-        private volatile int _statusDotNotCallMeDirectly; //0 by default
```

Comment:
Either use the `volatile` keyword or use `Volatile.Read` / `Volatile.Write` consistently, but don't do both.